### DISCUSSION TITLE
Cannot duplicate frame for imported single images (png/jpg)

### DISCUSSION THREAD

**Discussion by FerrLui** [opentoonz/opentoonz#6504] [Q&A]
### Steps to Reproduce
Create raster level, draw something, try to duplicate the drawing

### Expected Results-
Dublicate the drawing

### Actual Results-
Blank frame

### System Information
* **OpenToonz Version**:1.14 
* **Operating System**: Windows7
* **CPU**: Not metter
* **Memory**: Not metter
* **Graphics Card**: Nvidia
* **Graphics Tablet**: Not metter

**Comment by FerrLui** [comment]
i Dont know why, but it start to work, then again start show bugs...(

**Comment by RodneyBaker** [comment]
Are you sometimes using a shortcut/hotkey to duplicate the frame?

Alt + D creates a new empty frame whereas D is used for duplicating the currently selected frame.

**Comment by FerrLui** [comment]
I use both versions: 1. RMB -》 DUBLICATE DRAWING
2. AND D key. 
 YOU CAN SEE IT ON VIDEO. I USE D KEY FOR DUPLICATE WITHOUT ALT

**Comment by RodneyBaker** [comment]
Very strange.
Perhaps someone can confirm but I can't reproduce the problem on my end.
I'm on Win10 so perhaps it relates to Windows 7.

**Comment by gab3d** [comment]
I couldn't either (Win 10)

**Comment by FerrLui** [comment]
Really, maybe  problem only on windows7?

**Comment by manongjohn** [comment]
> Really, maybe problem only on windows7?

Doubtful. Something like that is not platform specific. I looked at your video but it was hard to tell since you didnt have a level strip open to show what was being duplicated from xsheet and what the new frame made. It appeared to have created a new frame though it seems the contents did not get copied?

**Comment by manongjohn** [comment]
For the record, I was able to duplicate successfully on a Win7 machine.

**Comment by FerrLui** [comment]
Okey, Iunderstand where is the problem. Try to import image. And then duplicate it,
[In this video](https://youtu.be/t3EdC3l-msI) u can see :
 At first im creating raster level, draw something, duplicate it, all working.
 Then im importing image, try to dublicate, Bug, Blank frame

**Comment by RodneyBaker** [comment]
I am seeing something upon trying to duplicate an imported/loaded image.
This may relate to the read only nature of specific file types.
Not sure.  More investigation required.

**Comment by RodneyBaker** [comment]
As a test/workaround can you create a new empty raster level (not toonz raster) and then copy/paste the imported image into that new level?

You should be able to duplicate that 'proper' level without issue.

**Comment by FerrLui** [comment]
That way all is working

**Comment by manongjohn** [comment]
Confirmed the issue using loaded single image jpg file as well as a single image png file.

**Comment by RodneyBaker** [comment]
Confirmed to still be an issue in current v1.6 nightly releases.

The underlying problem appears to relate to the special handling for single images (if not also for those treated as read only).

**Comment by RodneyBaker** [comment]
As this is standard operations for single images will likely move this to Opentoonz - Docs
